torch/ao/sparsity/experimental/pruner/base_pruner.py

```diff
@@ -166,13 +176,18 @@ def convert(self, use_path=False, *args, **kwargs):
                 del module._buffers['mask']
             delattr(module, 'mask')
 
+    def convert(self):
+        # TODO: Call the torch.ao.utils.convert in here
+        raise NotImplementedError('`convert` is not implemented. Please, use '
+                                  '`torch.ao.utils.convert` instead.')
+
```

Comment:
Hoping that most of the methods dont need to be written again and can be just inherited.